sim-cli/src/parsing.rs

```diff
@@ -8,7 +8,7 @@ use simln_lib::{
     ActivityDefinition, Amount, Interval, LightningError, LightningNode, NodeId, NodeInfo,
     Simulation, SimulationCfg, WriteResults,
 };
-use std::collections::HashMap;
+use std::collections::{HashMap, HashSet};
```

Comment:
nit: commit message should wrap at 72 chars, here and elsewhere in the PR

see [docs](https://github.com/bitcoin-dev-project/sim-ln/blob/main/docs/CONTRIBUTING.md#commit-history)